scio-core/src/main/scala/com/spotify/scio/values/PairSCollectionFunctions.scala

```diff
@@ -353,6 +352,39 @@ class PairSCollectionFunctions[K, V](val self: SCollection[(K, V)])
     hotJoined ++ chillJoined
   }
 
+  /**
```

Comment:
No need to. `trait Hash128` has a companion `object Hash128` with common instances and those are in the default lookup scope. Nice little implicit trick.